tf2_ros/package.xml

```diff
@@ -27,7 +27,10 @@
   <exec_depend>tf2</exec_depend>
   <exec_depend>tf2_msgs</exec_depend>
 
+  <build_export_depend>message_filters</build_export_depend>
   <build_export_depend>rclcpp</build_export_depend>
+  <build_export_depend>tf2</build_export_depend>
+  <build_export_depend>tf2_msgs</build_export_depend>
```

Comment:
I would use `<depend>` for this four packages, since they are now used in `<build_depend>`, `<exec_depend>` and `<build_export_depend>`.

See http://docs.ros.org/jade/api/catkin/html/howto/format2/migrating_from_format_1.html.

Reply:
Done (5aebc0c)